demos-europe/demosplan-core
# feat (refs T29639): replace webatlas by basemap

**Ticket:** https://yaits.demos-deutschland.de/T29639

Description: This PR includes a core migration to replace the webatlas with basemap.

### How to review/test
- execute migration
- check if in all procedure templates webatlas is replaced by basemap
(Verwalten -> Blaupausen -> select one -> Planungsdokumente und Planzeichnung -> Kartenebenen definieren)
- to check if the master template is also changed, create a new procedure template
- to check if the copyright is changed (Verwalten -> Blaupausen -> select one -> Planungsdokumente und Planzeichnung -> Startkartenausschnitt festlegen)
- check the database (tables: _gis, _procedure_settings and customer)

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
